### ISSUE TITLE
Request: ability to use double quotes

### ISSUE THREAD

**Issue by jamesbraza** [jumanjihouse/pre-commit-hook-yamlfmt#53]
Python `black` prefers double quotes: https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html#strings

Since I use Python, I like my YAML to use double quotes too.  Running this tool on my repo moves it to use single quotes:

```none
-        if: "github.event_name == 'release' && github.event.action == 'published'"
+        if: github.event_name == 'release' && github.event.action == 'published'
```

There is the `--preserve-quote`s option, which leads to not touching quotes at all.  I actually like when this tool makes changes like this, so I want to keep `--preserve-quote` on:

```none
-                  cache: "pip"
+                  cache: pip
```

My request is to add a new option resembling `black`'s `--skip-string-normalization` that will just not convert double quotes to single quotes.

**Comment by aeisenbarth** [comment]
To me, black's description for `--skip-string-normalization` sounds analog to `--preserve-quotes`:
> If this option is given, strings are left unchanged instead.

Is that your request, or rather an option to enforce a certain type of quotes?
For my project, I wanted to enforce double quotes because:
1. unquoted strings in YAML can cause unexpected behavior (`no → bool`, trailing spaces being included in strings…) and
2. the preferred type of quotes depends on the context and should be configurable.

How should such an option look like?
- `--preserve-quotes` |  `--single-quotes` | `--double-quotes`
- `--quotes="preserve"` | `--quotes="single"` | `--quotes="double"`
- `--quotes-style=…`

**Comment by jamesbraza** [comment]
Yeah @aeisenbarth, thanks for the follow up question. I agree in hindsight my OP was a little vague. I am interested in the ability to format with double quotes as opposed to single quotes. I think your proposed options `--double-quotes` or `--quotes=double` look great
